### ISSUE TITLE
Add alternative inflation measures to the dataset

### ISSUE THREAD

**Issue by matdehaven** [VFCI/vfciBusinessCycles#79]
Currently we use the GDP deflator in the VAR to measure inflation.

Instead add options for

- [x] CPI
- [x] CPI ex Food and Energy
- [x] PCE
- [x] PCE ex Food and Energy

**Comment by matdehaven** [comment]
- [ ] Create report comparing the different inflation measures and how the affect the baseline BCA result.

**Comment by matdehaven** [comment]
- [ ] Move inflation data to new R script that pulls series directly from FRED for use in the project

Should move some of the other indexes to this function as well.  No need to have them all go through the VFCI construction script (this script limits their date ranges for instance).

**Comment by matdehaven** [comment]
@fernando-duarte 

[Report comparing different inflation measures](https://github.com/VFCI/vfciBusinessCycles/blob/06b52c2ead41e7c1ca64007058cbf9fc8db96a0e/reports/qmds/compare-inflation-measures.pdf)

This report compares different inflation measures (GDP deflator, pce, CPI, PCE ex food and energy and CPI ex food and energy) and if there is any impact on the BCA max share VAR estimation.

There is a small difference.

- The IRFs are larger for Inflation's response to the BCA shock when using CPI and CPI ex F&E.  About twice as big.  Negligible differences when using PCE or PCE ex F&E.
- However, the forecast error variance decomposition for the BCA shock is still small for inflation---16%, 28%, 13%, 15%---compared to ~17% when using the gap deflator.  The 28% is from using the CPI ex F&E.
- The responses of the rest of the variables hardly move at all.
- The business cycle shock identified with the new inflation measures are correlated 0.93, 0.98, 0.98, and 0.99 with the original gdp deflator business cycle shock.

In conclusion, I don't think we should worry about the inflation measure.  We can now run robustness on the different measures for our final specifications, but I doubt we will see noticeable differences.
